components/PageNavigation.vue

```diff
@@ -75,12 +80,11 @@
         </li>
       </template>
       <li
-        v-else-if="enableLoginLink"
```

Comment:
Removing this conditional has caused a regression. When I log in the log in button remains in the desktop navigation and both log in and log out are shown in the mobile navigation

<img width="1294" alt="Screenshot 2020-11-18 at 21 32 47" src="https://user-images.githubusercontent.com/4670058/99533982-0d783580-29ea-11eb-8ca8-9f20021949bf.png">

<img width="1294" alt="Screenshot 2020-11-18 at 21 34 26" src="https://user-images.githubusercontent.com/4670058/99533973-094c1800-29ea-11eb-8aba-2192d97b5d68.png">